app/src/main/java/com/kickstarter/ui/fragments/DiscoveryFragment.java

```diff
@@ -44,7 +43,8 @@
 
   public DiscoveryFragment() {}
 
-  public static @NonNull Fragment newInstance(final int position) {
+  @NonNull
```

Comment:
I think the auto-formatter messed this up as I didn't specifically change this. Is it possible to change the settings file so this doesn't happen on auto-format?